src/__stories__/meter-story.tsx

```diff
@@ -100,6 +110,21 @@ export const MeterStory: StoryComponent<MeterStoryArgs> = ({
                         reverse={reverse}
                         values={values}
                         width={fullWidth ? '100%' : width}
+                        {...(extraContent
+                            ? {
+                                  extra: (
+                                      <div
+                                          style={{
```

Comment:
would it make sense to use <Align> of mistica here instead of css?

Reply:
As it is a story inside mistica, I prefer to have stories isolated from other components.

Reply:
take into account this story enables screenshot tests, I think these tests should not fail when, for example, a box is changed in mistica.